analyzers/src/SonarAnalyzer.CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -135,9 +135,9 @@ public static ControlFlowGraph CreateCfg(this SyntaxNode body, SemanticModel sem
                         cfg = cfg.GetAnonymousFunctionControlFlowGraph(flowOperation);
                         cfgFlowOperations = cfg.FlowAnonymousFunctionOperations();
                     }
-                    else if (node == body)  // 'body' should always reach  LocalFunction CFG or AnonymousFunction CFG above
+                    else if (node == body)
                     {
-                        throw new InvalidOperationException($"Could not find CFG for {body.Parent.Kind()} in {body.SyntaxTree.FilePath}:{body.GetLocation().GetLineNumberToReport()}");
+                        return null;
```

Comment:
Aligning implementation with VB.NET to be on the safe side. I couldn't come up with a properly invalid syntax to reach this path. Not even just this in a file:
```
() =>
```